packit_service/worker/build/copr_build.py

```diff
@@ -185,7 +192,137 @@ def monitor_not_submitted_copr_builds(self, number_of_builds: int, reason: str):
                 reason=reason
             ).observe(time)
 
+    def get_packit_stable_copr_download_urls(self) -> List[str]:
+        """
+        Get packit package download urls for latest succeeded build in packit/packit-stable project
+        e.g .https://download.copr.fedorainfracloud.org/results/packit/
+        packit-dev/fedora-35-x86_64/02996411-packit/
+        python3-packit-0.42.1.dev8+g5f7e0b0-1.20211201073310374840.main.8.g5f7e0b0.fc35.noarch.rpm
+
```

Comment:
This is becoming a bit recursive and meta...;)

Can we make the project configurable so it's possible to use a different project for production and stage?

Reply:
> This is becoming a bit recursive and meta...;)

Yeah, I am open to other options on how to have the RPM for packit stable available, but currently this option seems the easiest to me.

> Can we make the project configurable so it's possible to use a different project for production and stage?

Oh, sure, thanks for pointing that out!

Reply:
> Yeah, I am open to other options on how to have the RPM for packit stable available, but currently this option seems the easiest to me.

No, that's really nice! And with Packit, we can easily have a different project for a different branch.